Задание для кандидатов. Выполнить ревью.sql

```diff
@@ -0,0 +1,123 @@
+create procedure syn.usp_ImportFileCustomerSeasonal
+	@ID_Record int
+AS
+set nocount on
+begin
+	declare @RowCount int = (select count(*) from syn.SA_CustomerSeasonal)
+	declare @ErrorMessage varchar(max)
+
+-- Проверка на корректность загрузки
+	if not exists (
+	select 1
+	from syn.ImportFile as f
+	where f.ID = @ID_Record
+		and f.FlagLoaded = cast(1 as bit)

```

Comment:
Комментарий 16
Алиасы называют тремя буквами таблицы при возможности
```suggestion
	from syn.ImportFile as imf
	where imf.ID = @ID_Record
		and imf.FlagLoaded = cast(1 as bit)

```